doc/import_production_data.md

```diff
@@ -0,0 +1,23 @@
+# Import Production Data
+
+The content_performance_manager database is no longer imported by default.
+
+To import production data, first either download the data as documented in 
+[the GOV.UK developer docs](https://docs.publishing.service.gov.uk/manual/get-started.html#7-import-production-data)
```

Comment:
The link seems to be wrong.
Could you please detail what instructions need to be run in order to download a database that is not downloaded by default?